corehq/apps/hqadmin/migrations/0001_initial.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
```

Comment:
@emord should i just move this to a `south_migrations` folder? last time it did it automatically for me, but it wasn't the initial migration